### PR TITLE
Add new licenses

### PR DESCRIPTION
**Pull request by akugarg**
This PR new licenses

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: src/licensedcode/data/licenses/dpl.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+key: dpl
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/dpl.yml]: Since this is versioned as 1.1, please rename (here and also the filenames) as `dpl-1.1`

Comment (akugarg) [reply src/licensedcode/data/licenses/dpl.yml]: There exists already a license with this key https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/dpl-1.1.yml

**Comment by akugarg** [comment]
@pombredanne the failing check mentioned here ,is it related to something that I have added?

File: src/licensedcode/data/licenses/dpl.yml (hunk 1 above)
Comment (pombredanne) [reply src/licensedcode/data/licenses/dpl.yml]: name it with this key then  `defensive-patent-1.1`

File: src/licensedcode/data/licenses/dpl.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+short_name: DPL
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/dpl.yml]:
```suggestion
short_name: Defensive Patent License v1.1 
```

File: src/licensedcode/data/licenses/dpl.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+key: dpl
+short_name: DPL 
+name: THE DEFENSIVE PATENT LICENSE v1.1
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/dpl.yml]:
```suggestion
name: Defensive Patent License v1.1
```

File: src/licensedcode/data/licenses/dpl.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+owner: Unspecified
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/dpl.yml]: It must have an owner

**Review by pombredanne — CHANGES_REQUESTED** [review]

**Comment by pombredanne** [comment]
> @pombredanne the failing check mentioned here ,is it related to something that I have added?

Yes and no... I think we could remove entirely that test failing tests (tests/licensedcode/test_query.py::TestQueryWithFullIndex::test_Query_tokens_by_line_behaves_the_same_on_various_python ) and its data files as this only served a purpose when we were porting from Python 2 to Python 3.

Nowadays it only ever fails when we have some significant new words added in new licenses... and that's useless since we are all on Python 3

**Review by pombredanne — APPROVED** [review]
LGTM! Thanks and merging.

**Pull request merged**